.github/ISSUE_TEMPLATE/5_cdn.yml

```diff
@@ -0,0 +1,51 @@
+name: CND Issues
+description: Use this template to report issues with the CDN.
+labels:
+  - type::hosting
+body:
+  - type: textarea
+    id: channels
+    attributes:
+      label: Channel(s)
+      description: List the affected channel(s) (e.g., conda-forge, bioconda).
+    validations:
+      required: true
+  - type: input
+    id: date
+    attributes:
+      label: When
+      description: Provide when the problem was first noticed.
+    validations:
+      required: true
+  - type: dropdown
+    id: type
+    attributes:
+      label: Problems
+      description: Select the type of problem you are reporting.
+      multiple: true
+      options:
+        - repodata completeness
+        - package download speed
+        - channel availability / accessibility
+        - other
+    validations:
+      required: true
+  - type: textarea
+    id: packages
+    attributes:
+      label: Packages
+      description: Example packages or URLs to investigate.
+  - type: textarea
+    id: curl_output
+    attributes:
+      label: Output of `curl -I EXAMPLE_URL`
```

Comment:
We probably would want to include `-sSL` as options for `curl` (so errors are displayed and any redirects followed).

Reply:
```suggestion
      label: Output of `curl -sSLI EXAMPLE_URL`
```